api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterByNode.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 - 2019 Valtech GmbH
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package de.valtech.aecu.api.groovy.console.bindings.filters;
+
+
+import javax.annotation.Nonnull;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.resource.Resource;
+
+
+/**
+ * Filters resources by absolute or relative node path. Only resources that exist in the repository are accepted.
+ *
+ * @author Vugar Aghayev
+ */
+public class FilterByNode implements FilterBy {
+
+    private String path;
+
+    /**
+     * Constructor
+     *
+     * @param path node path
+     */
+    public FilterByNode(@Nonnull String path) {
+        this.path = path;
+    }
+
+    @Override
+    public boolean filter(@Nonnull Resource resource, StringBuilder output) {
+        if(StringUtils.isBlank(path))
+            return true;
```

Comment:
use brackets for all if-blocks
recheck if the formatter rules were correctly imported. There should be a space after the "if"

Reply:
imported the Intellij formatter, now the code is formatted as expected.